### ISSUE TITLE
outputs is coming empty in the operations of interfaces

### ISSUE THREAD

**Issue by ashish1988anshul** [tliron/puccini#118]
HI ,

https://github.com/tliron/puccini/blob/main/examples/tosca/interfaces.yaml

when we are parsing & compiling , 

Then the output is missing in the operations. 

```
						"operations": {
							"maintenance_off": {
								"dependencies": [],
								"description": "Turn maintenance mode off",
								"implementation": "/opt/scripts/maintenance_off.sh",
								"inputs": {
									"priority": 0.75
								},
								"outputs": {},
								"timeout": -1
							},
						},
```
Find the full file below
[interface.txt](https://github.com/tliron/puccini/files/10141219/interface.txt)

Expectation is : It should show like below 
```
	"operations": {
							"maintenance_off": {
								"dependencies": [],
								"description": "Turn maintenance mode off",
								"implementation": "/opt/scripts/maintenance_off.sh",
								"inputs": {
									"priority": 0.75
								},
								"outputs": {
                                                                   "timestamp": {
										"nodeTemplateName": "server",
										"target": "last_turned_off",
										"targetType": "attribute"
									}
},
								"timeout": -1
							},
						},


```

**Comment by tliron** [comment]
Thank you for finding the issue! Do you want to submit a PR or wait for me to fix it?

**Comment by ashish1988anshul** [comment]
Hi, Are you planning to do any release soon?      
   then please include this fix

It will be helpful

**Comment by tliron** [comment]
Yes, I am hoping to release next week!

**Comment by ashish1988anshul** [comment]
That will help. 
Please include this fix as well.

**Comment by ashish1988anshul** [comment]
Hi @tliron , I have raised a PR, Please check  https://github.com/tliron/puccini/pull/119 and merge if all are ok.

**Comment by tliron** [comment]
Thanks! I have a few divergences to fix in the main branch, I hope to finish this weekend!

**Comment by ashish1988anshul** [comment]
Cool!

**Comment by Shishqa** [comment]
> Hi @tliron , I have raised a PR, Please check #119 and merge if all are ok.

Hi!
I've noticed that there is the same issue in relationship operations. Tried to fix the issue the same way it was fixed in the PR and came across the issue of not validating SOURCE and TARGET in output mappings.

https://github.com/tliron/puccini/blob/3289a4489ea409101d28cd8c42d7bfb6057566a2/tosca/grammars/tosca_v2_0/output-mapping.go#L20-L22
Am I wrong or this statement is incomplete?
